app/src/renderer/vuex/modules/user.js

```diff
@@ -71,13 +71,9 @@ export default ({}) => {
     },
     async showInitialScreen({ state, dispatch, commit }) {
       dispatch(`resetSessionData`)
-
       await dispatch(`loadAccounts`)
       commit(`setModalSessionState`, `welcome`)
-
-      state.externals.track(`pageview`, {
-        dl: `/session/welcome`
-      })
+      state.externals.track(`pageview`, { dl: `/` })
```

Comment:
what is the thinking here?

Reply:
the main page is not longer `/session/welcome`